script/lib/lint-coffee-script-paths.js

```diff
@@ -9,7 +9,7 @@ const CONFIG = require('../config');
 
 module.exports = function() {
   const globPathsToLint = [
-    path.join(CONFIG.repositoryRootPath, 'dot-atom/**/*.coffee'),
+    path.join(CONFIG.repositoryRootPath, 'templates/**/*.coffee'),
```

Comment:
```suggestion
    path.join(CONFIG.repositoryRootPath, 'dot-atom/**/*.coffee'),
```
Revert this line to coincide with reverting templates -> dot-atom